docs/workflows/manage.rst

```diff
@@ -3,14 +3,26 @@ Manage Content
 
 Package
 -------
-
+Package (RPM or SRPM) is uniquely identified by NEVRA.
+Within a repository version there can be only one package with the same NEVRA.
+There can't be two packages with same NERVA and different checksum types within a repository version.
 
 Advisory
 --------
 
 
 Modularity Metadata
 -------------------
+RPM plugin will parse a modules.yaml metadata file into modules and modules-defaults.
+
+Modules are uniquely identified by NSVCA which stands for name, stream, version, context, and
+architecture. Modules contain artifacts (RPM packages), whenever a module is being added or copied
+to a repository, modules' packages are carried over. In case a module is removed from a repository
+version, modules' packages are removed as well, this way repository version  will not contain
+ursine (module unreferenced) packages and module consistent will be preserved.
+
+Modules-defaults are uniquely identified by NS which stands for name and stream.
```

Comment:
That's not necessarily true. It's unique by digest which we recalculate. Maybe we can remove this sentence completely, it's not important probably.